theme/hesk3/customer/view-ticket/view-ticket.php

```diff
@@ -60,6 +60,7 @@
                 <div class="header__inner">
                     <a href="<?php echo $hesk_settings['hesk_url']; ?>" class="header__logo">
                         <?php echo $hesk_settings['hesk_title']; ?>
+                        <img src="./img/favicon/simbolo.svg" alt="" width="25px" height="25px" style="vertical-align: -3px;">
```

Comment:
Seria interessante usar uma URL absoluta já que o Hesk não resolve URLs de assets.
Usar `./img...` pode fazer com que em algumas páginas as imagens apareçam quebradas ou seja, sem imagem.

```suggestion
                        <img src="<?php echo $hesk_settings['hesk_url']; ?>/img/favicon/simbolo.svg" alt="" width="25px" height="25px" style="vertical-align: -3px;">
```

Reply:
@rodrigopepato